### ISSUE TITLE
Exam Links in Exams Page

### ISSUE THREAD

**Issue by achen01** [haolinnie/Self-Service-Database-UI#13]
The page crashes when exam link is clicked on. Potentially link to patient images page? However, patient images page contain multiple exams and not just one.

**Comment by keremguventurk3123** [comment]
let's talk about what this button should point to, right now I made it not go to any page

**Comment by achen01** [comment]
Agreed, let's talk about this perhaps during tomorrow's meeting. Sounds good.
